versionmanager/retriever/terragrunt/terragruntretriever.go

```diff
@@ -103,12 +104,12 @@ func (r TerragruntRetriever) InstallRelease(versionStr string, targetPath string
 		return err
 	}
 
-	err = os.MkdirAll(targetPath, 0755)
+	err = os.MkdirAll(targetPath, 0o755)
```

Comment:
why is it 0o755?

Reply:
It's automatically changed on save in vscode. See more https://github.com/golang/go/issues/33363.